docs/api.html

```diff
@@ -78,6 +78,19 @@ <h3><a id="streamsapi" href="#streamsapi">2.3 Streams API</a></h3>
 		&lt;/dependency&gt;
 	</pre>
 
+	<p>
+	When using Scala you may optionally include the <code>kafka-streams-scala</code> library.  Additional documentation on using the Kafka Streams DSL for Scala is available <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html#scala-dsl">here</a>.
+	<p>
+	To use Kafka Streams DSL for Scala you can use the following maven dependency:
+
+	<pre class="brush: xml;">
+		&lt;dependency&gt;
+			&lt;groupId&gt;org.apache.kafka&lt;/groupId&gt;
+			&lt;artifactId&gt;kafka-streams-scala&lt;/artifactId&gt;
+			&lt;version&gt;{{fullDotVersion}}&lt;/version&gt;
+		&lt;/dependency&gt;
```

Comment:
I am wondering if this is correct? Should the Scala version not be included here?

Reply:
I'm not very familiar with gradle, but it appears to not support cross building jars in the same manner as SBT.  The build needs to be run for each Scala version you want a jar for, but the output won't encode the version into the filename.  I think what we need to do is add a task to the gradle file, or some other build related packaging script, to pluck the generated `kafka-streams-scala` file, rename it to include the Scala version, and then publish it to maven central.

Ex) The built outputs this when specifying a 2.12 `SCALA_VERSION` (`./gradlew -PscalaVersion=2.12 jar`)

```
-rw-rw-r--  1 seglo seglo 105423 Apr 18 14:17 kafka-streams-scala-1.2.0-SNAPSHOT.jar
```

When a release artifact is published we'll publish a file: `kafka-streams-scala_2.12-1.2.0.jar` with the Scala major version encoded into the artifact name.  A maven user would reference the artifact with:

```
<dependency>
    <groupId>org.apache.kafka</groupId>
    <artifactId>kafka-streams-scala_2.12</artifactId>
    <version>1.1.0</version>
</dependency>
```
Please let me know if I'm missing something here about the Kafka build system.

On a related note I found a gradle build plugin that handles cross building projects and referencing Scala dependencies in a SBT style here: https://github.com/ADTRAN/gradle-scala-multiversion-plugin

Reply:
We already so the right thing for core jars. We just need to follow the same approach. And yes, the Scala version needs to be encoded in the artifact id. Not sure what @guozhangwang was trying to say, but doesn't seem correct to me.

Reply:
@ijuma Ok, is the approach you refer to in the core project of the `build.gradle`?  I'll take a closer look.  

WRT the docs @mjsax is correct that we should update the maven dependency examples to include the Scala version.

Reply:
What I was saying is that when we build the artifact we already chose which scala vertion to use compiling the jar and made the scala version as part of the artifact name, so users do not need to specify the scala version in declaring the dependency, but just:

```
<dependency>
    <groupId>org.apache.kafka</groupId>
    <artifactId>kafka_2.12</artifactId>
    <version>1.1.0</version>
</dependency>
```

Reply:
I'll correct this to include the Scala version (2.11) across all the maven `pom.xml` references.